spec/unit/user_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'user'
+
+describe User do
+  describe '.create' do
+    it 'hashes the password using BCrypt' do
+      expect(BCrypt::Password).to receive(:create).with('password123')
+      User.create('test1@example.com', 'password123', 'test1', 'test1')
+    end
+
+    it 'creates a new user' do
+      user = User.create('test2@example.com', 'password123', 'test2', 'test2')
+      # persisted_data = persisted_data(user, user.id)
+      expect(user).to be_a User
+      # expect(user.id).to eq persisted_data.first['id']
```

Comment:
You can probably delete the commented-out lines 12 and 14 - I'm guessing they are left over from an earlier test. :)

Reply:
yes, forgot to remove 😂